### ISSUE TITLE
Cannot turn on accessibility access for zenith 

### ISSUE THREAD

**Issue by STEVE123132** [1372Slash/Zenith#51]
i updated the app to v1. 5.7 then it just wont let me enable accessibility access to zenith it says my google wallet is a banking app and wont let me enable it it will just send a notification and disable it

**Comment by 1372Slash** [comment]
same issue as #48. resolved on 62d5485b20ec64f90cf6a3da85864522800a9736
